<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QParameterEditor</class>
 <widget class="QWidget" name="QParameterEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Parameter Editor</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <widget class="QLabel" name="valueInvalidLabel" >
     <property name="toolTip" >
      <string>The value must be a number (e.g. "pi^2")</string>
     </property>
     <property name="text" >
      <string>(invalid)</string>
     </property>
    </widget>
   </item>
   <item rowspan="6" row="1" column="0" >
    <widget class="QListWidget" name="list" />
   </item>
   <item row="1" column="1" >
    <widget class="KPushButton" name="cmdNew" >
     <property name="toolTip" >
      <string>Add a new constant</string>
     </property>
     <property name="whatsThis" >
      <string>Click this button to add a new constant.</string>
     </property>
     <property name="text" >
      <string>&amp;New</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="KPushButton" name="cmdDelete" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="toolTip" >
      <string>delete selected constant</string>
     </property>
     <property name="whatsThis" >
      <string>Click here to delete the selected constant; it can only be removed if it is not currently used by a plot.</string>
     </property>
     <property name="text" >
      <string>&amp;Delete</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>51</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" >
    <widget class="KPushButton" name="cmdImport" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="toolTip" >
      <string>Import values from a textfile</string>
     </property>
     <property name="whatsThis" >
      <string>Import values from a textfile. Every line in the file is parsed as a value or expression.</string>
     </property>
     <property name="text" >
      <string>&amp;Import...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="KPushButton" name="cmdExport" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="toolTip" >
      <string>Export values to a textfile</string>
     </property>
     <property name="whatsThis" >
      <string>Export values to a textfile. Every value in the parameter list will be written to one line in the file.</string>
     </property>
     <property name="text" >
      <string>&amp;Export...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>71</width>
       <height>281</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" >
    <widget class="EquationEdit" name="value" />
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>EquationEdit</class>
   <extends>KLineEdit</extends>
   <header>equationedit.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends></extends>
   <header>klineedit.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends></extends>
   <header>kpushbutton.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
